proto/records/genesis.proto

```diff
@@ -59,6 +59,8 @@ message DepositRecord {
     TRANSFER = 0;
     // pending staking on delegate account
     STAKE = 1;
+    // ICA call is pending
```

Comment:
it would be nice to differentiate whether it's pending from an IBC transfer or delegation ICA. Maybe PENDING_TRANSFER and PENDING_STAKE?

This also might not be a bad time to re-evaluate these status fields in general. TRANSFER and STAKE are a little unintuitive.

Reply:
Renamed